### DISCUSSION TITLE
Feature Request: Filterwheel control

### DISCUSSION THREAD

**Discussion by AdityaKinjawadekar** [aaronwmorris/indi-allsky#1016] [Ideas]
Sorry if it has been requested before or it already exists, but I wanted to request a feature of supporting filterwheel in an all-sky camera. I am building a system with ASI533, a fisheye lens, and I am incorporating a filter wheel to measure photometry-based extinction in V, B, and R filters. I wanted to rotate the filter between two consecutive images to perform photometry in a particular band. 

If not an added feature (which I understand very few people will actually use it), I would like to know how I can write my own script to do so. Any inputs regarding the same are welcome :)

**Comment by aaronwmorris** [comment]
Writing a python script to manage a filter wheel should not be too difficult.  I may be able to write an example over the next few days.

There is not really a way to integrate this functionality directly into indi-allsky currently.  I am planning on adding a "plugin" system to indi-allsky that could introduce hooks into different stages of the workflow, but this is very conceptual right now.
